package.json

```diff
@@ -9,18 +9,20 @@
     "src"
   ],
   "scripts": {
+    "#postinstall": "postinstall hook is required to 'fix' ava d.ts file",
+    "postinstall": "rimraf node_modules/ava/index.d.ts",
```

Comment:
Fair enough. Couldn't we do this?
https://www.typescriptlang.org/docs/release-notes/typescript-1.8.html#augmenting-globalmodule-scope-from-modules